azkaban-webserver/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -338,6 +340,24 @@ private void handleExecutionFlowPage(HttpServletRequest req,
       page.render();
       return;
     }
+    
+    Props props = getApplication().getServerProps(); 
+    String execExternalLinkURL = 
+        props.getString("execution.external.link.url", "");
+    String execExternalLinkLabel = 
+        props.getString("execution.external.link.label", "External Analyzer");
+        
+    if (StringUtils.isNotEmpty(execExternalLinkURL)) {
+      StringBuilder builder = new StringBuilder();
+      builder.append(req.getRequestURL());
+      builder.append("?");
+      builder.append(req.getQueryString());
+      String flowExecutionURL = builder.toString();
```

Comment:
I agree with your point that only execution ID is enough and is cleaner.

However, I will have to check with the Dr. Elephant team on this. If it's too much work on Dr. Elephant side, will it be ok to have it like this for a while? Please let me know.